src/main/cpp/t05_power.cpp

```diff
@@ -27,9 +27,23 @@
 
 using namespace std;
 
-//function power
+double power(double a, long n){
+    if (n == 0){
+        return 1;
+    }
+    return a*power(a, --n);  
+}
 
 int t05_power() {
-
+    double a = 0;
+    long n = 0;
+    cin >> a >> n;
+    if (n >= 0)
+    {
+        cout << power(a, n) << endl;
+    }
+    else if (n < 0){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Condition 'n<0' is always true](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4594063)